week01/assignment.go

```diff
@@ -0,0 +1,114 @@
+package main
+
+import (
+	"bufio"
+	"errors"
+	"fmt"
+	"os"
+	"strconv"
+	"strings"
+)
+
+func isOperator(text string) (op string, err error) {
```

Comment:
Better name: `isValidOperator` and your function don't need to return an error, boolean is good enough.